index.html

```diff
@@ -1879,6 +1786,99 @@ <h2 id="AudioContext">
             </p>
           </dd>
           <dt>
+            Promise&lt;void&gt; suspend()
```

Comment:
Yes. This is AudioContext.suspend() and the behavior has not changed except for its membership from BAC to AC.